pom.xml

```diff
@@ -142,6 +142,58 @@
 
     <build>
         <plugins>
+            <plugin>
+                <groupId>io.confluent</groupId>
+                <version>0.9.0-SNAPSHOT</version>
```

Comment:
Why are we using a snapshot release here?

Reply:
Agh, forgot to change that. Important to note that we shouldn't actually use the `0.9.0` release when this is actually merged though, since it generates incorrectly-formatted plugin archives (see https://github.com/confluentinc/kafka-connect-packaging-plugins/pull/25).